src/Connector.sol

```diff
@@ -139,15 +150,33 @@ contract CentrifugeConnector is Test {
         memberlist.updateMember(user, validUntil);
     }
 
-    function transferTo(
+    function handleTransfer(
         uint64 poolId,
         bytes16 trancheId,
         address user,
         uint256 amount
     ) public onlyRouter {
         RestrictedTokenLike token = RestrictedTokenLike(tranches[poolId][trancheId].token);
+        require(address(token) != address(0), "CentrifugeConnector/unknown-token");
         require(token.hasMember(user), "CentrifugeConnector/not-a-member");
```

Comment:
Do we have a test for this already, i.e. testing that a transfer to a non-whitelisted address fails? I think this check is actually redundant as the underlying erc20 token already checks that the destination address is a member.

Reply:
Bumping this ^ @NunoAlexandre @AStox

Reply:
Minting doesn't check the memberlist. This was the change I had made to `restricted.sol`, since it seems kind of odd that a permissioned token can mint to any address

Reply:
Gotcha. I would keep it like this for now, to not change the existing restricted token implementation. But let's discuss this a bit more and we can still change it in V2 optionally.